src/style/colors.ts

```diff
@@ -15,7 +15,11 @@ export type DesignToken = {semantic?: [string]; resource_paths?: [string]; toStr
 export type TokensOptions = {primaryColor: string};
 export type GetColorTintOptions = {avoidReverseOnDark?: boolean};
 export type GetColorByHexOptions = {validColors?: string[]};
-
+export type PaletteOptions = {
+  lightnessFix?: boolean;
+  saturationFix?: boolean;
```

Comment:
consider renaming to a more boolean name like `adjustSaturation`, `shouldAdjustSaturation`
Or also here, think how the user can pass a numeric value instead of a boolean where they can control the adjustment. If the user pass `undefined` you can treat it as false of course.

Reply:
Done